### ISSUE TITLE
[Bug]: old version of Jetty 9 with multiple CVEs used by Hop

### ISSUE THREAD

**Issue by pjfanning** [apache/hop#4462]
### Apache Hop version?

latest

### Java version?

n/a

### Operating system

Other

### What happened?

Dependabot is creating PRs but they are being closed without reasons being provided.

Example
https://github.com/apache/hop/pull/4432

CVE-2024-9823
CVE-2024-8184
CVE-2023-26048
CVE-2023-26049

### Issue Priority

Priority: 3

### Issue Component

Component: Other

**Comment by hansva** [comment]
You are right, I should have added a reason.
I have attempted to move our Jetty version past the current release but when doing so it causes our code to silently start failing. I haven't had the time yet to dig into the root cause. I did create a ticket https://github.com/apache/hop/issues/3924 to move to a newer Jetty version. The goal will be to move to Jetty 12 and move away from `javax` packages to the new `jakarta` packages.
